ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,11 +32,11 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void EndVisitTest() throws Exception {
+    public void endVisitTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
```

Comment:
`Waiting the page to load` ensures that the searched patient identifier matches with the created test patient identifier. Otherwise without` waiting the page to load`, some times the comparison is made between the searched patient identifier with any identifier (`randomly picked from the db pool`) of the existing patient data present in the  `db server instance` because the `search patient indexing criteria` that searches existing patient takes a bit longer to retrieve the `created test patient data` from the `db pool`